release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -474,9 +473,72 @@ The terminology for the `ip_whitelist` and `ip_blacklist` annotations have been
 
 [id="ocp-4-18-storage-editing-overprovisioning-ratio_{context}"]
 ==== Over-provisioning ratio update after LVMCluster custom resource creation
-
 Previously, the `thinPoolConfig.overprovisionRatio` field in the `LVMCluster` custom resource (CR) could be configured only during the creation of the `LVMCluster` CR. With this release, you can now update the `thinPoolConfig.overprovisionRatio` field even after creating the `LVMCluster` CR.
 
+[id="ocp-4-18-release-notes-storage-cifs-smb-csi-driver_{context}"]
+==== Persistent storage using CIFS/SMB CSI Driver Operator is generally available
+{product-title} is capable of provisioning persistent volumes (PVs) with a Container Storage Interface (CSI) driver for the Common Internet File System (CIFS) dialect/Server Message Block (SMB) protocol. The CIFS/SMB CSI Driver Operator that manages this driver was introduced in {product-title} 4.16 with Technology Preview status. In {product-title} 4.18, it is now generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc#persistent-storage-csi-smb-cifs[CIFS/SMB CSI Driver Operator].
+
+[id="ocp-4-18-release-notes-storage-secret-csi-driver_{context}"]
+==== Secret Store CSI Driver Operator is generally available
```

Comment:
Content looks good to me for Secret Store part. 
Double confirm "to mount multiple secrets, keys, and certificates stored" => "to mount multiple secrets, keys and certificates stored" would be better?

Reply:
We generally use the serial (aka Oxford comma).